modules/user/client-react/components/ProfileView.web.jsx

```diff
@@ -1,196 +1,669 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Helmet from 'react-helmet';
-import { Link } from 'react-router-dom';
-import { StripeSubscriptionProfile } from '@gqlapp/payments-client-react';
-import { translate } from '@gqlapp/i18n-client-react';
-import {
-  Card,
-  //  CardGroup,
-  CardText,
-  //  CardTitle,
-  PageLayout
-} from '@gqlapp/look-client-react';
-import { Row, Col, Divider, Icon, Button, Spin as Loader } from 'antd';
-import UserVerificationsComponent from './verification/UserVerificationsComponent';
-import ProfileHeadComponent from './components/ProfileHeadComponent';
-// import UsersCardComponent from './UsersCardComponent';
-import settings from '../../../../settings';
-import AddressCardComponent from './components/AddressCardComponent';
-import userCardData from '../helpers/userCardData';
+import React from "react";
+import PropTypes from "prop-types";
+import Helmet from "react-helmet";
+import { translate } from "@gqlapp/i18n-client-react";
+import { PageLayout } from "@gqlapp/look-client-react";
+import { Spin as Loader } from "antd";
+import settings from "../../../../settings";
 
 // To Do Abstract Out
 
 class ProfileView extends React.Component {
-  getUserId = () => {
-    let id = 0;
-    if (this.props.match) {
-      id = Number(this.props.match.params.id);
-    } else if (this.props.navigation) {
-      id = Number(this.props.navigation.state.params.id);
-    }
-    return id;
-  };
-
   render() {
-    const renderMetaData = t => {
+    const renderMetaData = (t) => {
       return (
         <Helmet
-          title={`${settings.app.name} - ${t('profile.title')}`}
+          title={`${settings.app.name} - ${t("profile.title")}`}
           meta={[
             {
-              name: 'description',
-              content: `${settings.app.name} - ${t('profile.meta')}`
-            }
+              name: "description",
+              content: `${settings.app.name} - ${t("profile.meta")}`,
+            },
           ]}
         />
       );
     };
+
     const { t } = this.props;
     const { currentUser, currentUserLoading } = this.props;
 
     if (currentUserLoading && !currentUser) {
       return (
         <PageLayout select="/profile">
           {renderMetaData(t)}
-          <Loader text={t('profile.loadMsg')} />
+          <Loader text={t("profile.loadMsg")} />
         </PageLayout>
       );
     } else if (currentUser) {
       return (
         <PageLayout select="/profile">
           {renderMetaData(t)}
 
-          <Row gutter={5}>
-            <Col xs={{ span: 24 }} lg={{ span: 16 }}>
-              <Card style={{ display: 'block' }}>
-                <h2
+          <main className="home-content-layout ant-layout-content" id="content">
```

Comment:
Pass prop type='home' in PageLayout and remove main tag, those classNames are builtin to PageLayout component.

Reply:
ok